### DISCUSSION TITLE
Is the `@screen` directive still recommended in Tailwind 3

### DISCUSSION THREAD

**Discussion by janko** [tailwindlabs/tailwindcss#7516] [General]
I occasionally see the `@screen` directive mentioned in the docs, but I don't see it being documented in "[Functions & Directives](https://tailwindcss.com/docs/functions-and-directives)", and didn't see it being deprecated either. Is it fine if we still use it, or should we use the `screen()` function instead?

**Comment by adamwathan** [comment]
Hey! We decided to encourage `screen()` instead because it plays nicer with other PostCSS plugins. Nesting plugins for example don't bubble `@screen` properly by default because they have no idea what it means.

No plans to remove it though 👍🏻
